symplyphysics/laws/waves/fully_destructive_interference_condition.py

```diff
@@ -1,31 +1,49 @@
+"""
+Fully destructive interference condition
+========================================
+
+The interference of two waves is said to be fully destructive when the amplitude of the
+resulting wave is zero, i.e. the two waves cancel each other out. In that case, they must
+be an odd number of half-cycles out of phase between each other.
```

Comment:
```suggestion
have an odd number of half-cycles of phase between each other.
```

Reply:
`they must be an odd number` sounds correct, but `they must be an odd number ... between` doesn't.